main.go

```diff
@@ -619,3 +607,26 @@ func configFilePath() string {
 	}
 	return filepath.Join(fp, "mark")
 }
+
+func setLogLevel(cCtx *cli.Context) error {
+	logLevel := cCtx.String("log-level")
```

Comment:
this should use strings.ToUpper to ensure that we're also matching "info" or "Info".

Reply:
done